src/components/App.jsx

```diff
@@ -5,7 +5,7 @@ import { translate } from '../plugins/preact-polyglot'
 import classNames from 'classnames'
 
 import Sidebar from './Sidebar'
-import Topbar from './Topbar'
+import Topbar from './TopBar'
```

Comment:
Oops, that seems to be a typo from me since I renamed the variable name before. Could you rename the TopBar.jsx to Topbar.jsx instead please? Thanks.